### PR TITLE
bg-transparent works in cards + color fixes

### PR DESCRIPTION
**Pull request by agessler**
Fixes Issue
https://infopark.slack.com/archives/C050WJC9YU8/p1727089632703809

See in WC "🌈all the colors!1!🌈"

### CODE REVIEW

**Review by dcsaszar — CHANGES_REQUESTED** [review]
* [ ] `dark-grey` is missing from your test content but has issues
* [ ] `middle-grey` has issues inside `light-grey`, `middle-grey`, `dark-grey`

image

**Comment by dcsaszar** [comment]
@agessler

* [ ] FYI Please also check out the new Tick list widget icon colors. They may need the same or similar adjustments.

**Comment by agessler** [comment]
@dcsaszar with latest commits all should work

File: src/assets/stylesheets/index.scss [hunk 1]
```diff
@@ -1156,7 +1178,7 @@ section:focus >,
 
 // -- general badges -- //
 
-.bg-middle-grey {
+.bagde.bg-middle-grey {
```
Comment (dcsaszar) [inline src/assets/stylesheets/index.scss:1181]: * [ ] Weird typo. Is this unused? FYI I also see two `.jr-list-bagde` selectors in the existing CSS.

**Review by dcsaszar — COMMENTED** [review]
Looks good so far.
See comment for a potential typo.
Also I added icon list widgets to the test working copy.
* [ ] Icon list colors often don't match a single icon color for some reason.

**Review by dcsaszar — CHANGES_REQUESTED** [review]
Icon lists look consistent now.
* [ ] But I see one more issue with white/light/grey boxes in dark boxes.

image


How about this (goes without `important`):

```scss
.card {
  &.bg-primary,
  &.bg-secondary,
  &.bg-dark-grey,
  &.bg-success,
  &.bg-info,
  &.bg-danger {
    .bi {
      color: #fff;
    }
  }

  &.bg-white,
  &.bg-middle-grey,
  &.bg-light-grey {
    .bi {
      color: var(--bs-primary);
    }
  }

  &.bg-transparent {
    box-shadow: none;
    color: inherit;
  }
}
```

**Review by dcsaszar — APPROVED** [review]

**Pull request merged**